src/test/scala/beam/agentsim/infrastructure/geozone/generator/WgsCoordinateGenerator.scala

```diff
@@ -0,0 +1,47 @@
+package beam.agentsim.infrastructure.geozone.generator
+
+import scala.util.Random
+
+import beam.agentsim.infrastructure.geozone.WgsCoordinate
+
+object WgsCoordinateGenerator {
```

Comment:
Previously I wrote some tests that used it, but I changed the code a lot that all the small tests were removed by the big one. Now it is not used and would not affect anything if is removed, but I just kept because I have intention to write more tests later